### ISSUE TITLE
icon scaling problems ?

### ISSUE THREAD

**Issue by ralosch** [SylEleuth/gruvbox-plus-icon-pack#36]
hello
i have icon 
![52 works not](https://github.com/SylEleuth/gruvbox-plus-icon-pack/assets/99989932/b5a71e9f-9bfe-4416-b665-81da67a42d9f)
![56 works](https://github.com/SylEleuth/gruvbox-plus-icon-pack/assets/99989932/2166b8dc-049f-4a33-9b9f-84f5df8e7a2d)
scaling problems under tuxedo os
=======
KWin version: 5.27.10
Qt version: 5.15.11
Qt compile version: 5.15.11
XCB compile version: 1.14

Operation Mode: X11 only

Thanks for the most beautiful icon set
Robert

**Comment by SylEleuth** [comment]
You have to better explain what is going so I could reproduce it.

**Comment by ralosch** [comment]
under kde i only see the correct icons in the control bar from a size of 56

**Comment by ralosch** [comment]
the usual program icons are also not visible in the application launcher. this has been the case for me since 5.0. Maybe the problem is with kde?
![Screenshot_20240103_215821](https://github.com/SylEleuth/gruvbox-plus-icon-pack/assets/99989932/9f5bbbdf-f9b7-491e-834c-e6da3ce6071f)

**Comment by SylEleuth** [comment]
I can't install TuxedoOS on VM because it requires registering to even download an iso so I hope plasma doesn't have many changes in comparison with regular clean one.

Also I'm not sure why so many icons don't have proper colours.

Bitwig is missing, I will add it tomorrow to repository. Same with Helvum. Not sure what kind of app is Tuxedo.

On the last screenshot, first app on the right. What is it?

Panel icons on the right are not from icon pack but from plasma desktop theme. I've already created it but because Plasma is going to drop it with version 6 and use regular icons from the icon packs i decided to not focus on it at the moment. Light version is sort of finished and even has proper icon colours but I don't use myself because it would require to have light gruvbox icons as well. But if You are using light theme You can try it anyway. You can find it [here](https://github.com/SylEleuth/gruvbox-plus-kde). There is no installation info but process is not difficult. Just copy/clone that folders to home/user/.local/share/plasma/desktoptheme/ and change to it in the settings.

**Comment by ralosch** [comment]
i can download it here without registering.

https://os.tuxedocomputers.com/

**Comment by ralosch** [comment]
> On the last screenshot, first app on the right. What is it?

vorta. Frontend for borg backup
https://vorta.borgbase.com/

**Comment by ralosch** [comment]
> You can find it [here](https://github.com/SylEleuth/gruvbox-plus-kde).

ups the link goes to 404

**Comment by ralosch** [comment]
https://www.tuxedocomputers.com/en/Test-TUXEDO-OS-in-VirtualBox-VMware-or-Gnome-Boxes.tuxedo#

**Comment by SylEleuth** [comment]
> > You can find it [here](https://github.com/SylEleuth/gruvbox-plus-kde).
> 
> ups the link goes to 404

Sorry, it's private repo. Look [here](https://github.com/SylEleuth/dotfiles/tree/master/KDE/.local/share/plasma/desktoptheme).

**Comment by SylEleuth** [comment]
Scaling icons was supposed to be a feature not a bug. I've created it for my personal usage and not fully tested it. I didn't even pushed it to the repo yet. Including it in the 5.0 version was my error and I've reverted it to the previous state. Some icons will be missing (because I have to make some changes while reversing) but added Bitwig (fixed actually) and Vorta. Didn't find Tuxedo app, but added distribution logo of that OS. The  pack has been updated to 5.0.1 version.
